doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -193,7 +194,29 @@ See :ref:`cmake_options` for more information about defining CMake options.
 Performing prerequisite operations
 **********************************
 
-You must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option to support the Google Fast Pair standard in your project.
+To support the Google Fast Pair subsystem in your project, you need to follow the steps in the following subsections that describe necessary Kconfig and API actions:
```

Comment:
```suggestion
To integrate the Google Fast Pair subsystem in your project, complete the following steps:
```

Reply:
adding word prerequisite as these steps are not sufficient for the whole integration (we have other steps in the integration guide):

```
To start integrating the Google Fast Pair subsystem in your project, complete the following prerequisite steps:
```